CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Thank You
+
+For taking an interest in Musicbox and thank you in advance for any contributions that you may make to its future.
+
+# Guidelines
+
+1.  Open a Pull Request
+    1.  Feel free to add yourself to this document!
+2.  Please make sure that your branch builds green
+3.  Merge away or wait for approval (depending on your access to the github repository itself)
+
+# Contributors
+
+- Dan Lavin
+- Jason Sisk
+- Sean Dennison
+- Truman Shuck
+
+# Source Available
```

Comment:
Does this section feel weird or wrong?  I can't decide if it's important to say or if it makes us greedy takey-takers.  I think that if this thing actually starts making money that we should cut off 10% revenue to contribute to the open source projects that we use somehow.

Reply:
If it reads weird, it's probably the legalese/language—not that the sentiment is wrong or greedy. Our primary objective is still to build a profitable product. Arguably, making the source available and free to use locally **is** an example of one such open source contribution. I'm thinking this section will just have to evolve if we ever get to the point where there are some really serious and frequent contributors.